plot.go

```diff
@@ -439,49 +439,71 @@ func (p *Plot) NominalY(names ...string) {
 	p.Y.Tick.Marker = ConstantTicks(ticks)
 }
 
-// Save saves the plot to an image file.  The file format is determined
-// by the extension.
+// WriterTo returns an io.WriterTo that will write the plot as
+// the specified image format.
 //
-// Supported extensions are:
+// Supported formats are:
 //
-//  .eps, .jpg, .jpeg, .pdf, .png, .svg, and .tiff.
-func (p *Plot) Save(w, h vg.Length, file string) (err error) {
+//  eps, jpg|jpeg, pdf, png, svg, and tif|tiff.
+func (p *Plot) WriterTo(w, h vg.Length, format string) (io.WriterTo, error) {
```

Comment:
An alternative would be to just return an io.WriterTo, but in the case of the error return:
type errWriterTo error
func (e errWriterTo) WriteTo(w io.Writer) (int64, error) { return 0, e }

Maybe it's a bit round-about, but I thought that I'd mention it.